Tools/autotest/arducopter.py

```diff
@@ -1919,6 +2023,9 @@ def autotest(self):
             '''vision position'''  # expects vehicle to be disarmed
             self.run_test("Fly Vision Position", self.fly_vision_position)
 
+            # Zigzag mode test
+            self.run_test("Fly ZigZag Mode", self.fly_zigzag_mode)
```

Comment:
Is there a way to make this function call depend on the `MODE_ZIGZAG_ENABLED` macro ?

Reply:
@amilcarlucas, I don't believe so, at least not directly.  Any concerns with the autotest?  is it too long?

Reply:
On our systems we comment out any features that we do not use, and we need to comment out the tests as well.
If there was a generic mechanism to import the C macros into python, this second "commenting-out step" could be avoided.

Reply:
@amilcarlucas no standard way to do it for now. We could make a regex parsing vehicle config.h to get which test are doable or not.

Reply:
That would be great, and could be used in a lot of places in the tests